_extras/guide.md

```diff
@@ -23,109 +27,127 @@ Because this lesson follows a single dataset throughout, its easy for learners (
+* It is recommended that at a minimum each day be a new notebook to avoid a single notebook that is so large that it become slow to interact with. While it would be great to demonstrate the best practice of breaking a complex analysis into multiple notebooks by starting each new episode in a new notebook, this will add overhead to your teaching time both to launch a new notebook and get it set up with the previously defined variables, so use your best judgement based on the level of your audience. If you decide to start each episode in a new notebook you can use the "Starting from this Episode" call out at the beginning of each episode to import and define everything you need from previous episodes. If you are using files from the `student_download` directory be sure to add the correct path `student_download/data/<filename>`.
 
-### Lesson 1: Queries
+* It is recommended that you do not repeat variable names and that you do not modify queries in place. If a student misses a step then they do not have an easy way to determine that they do not have the same code that you have.
 
-* At the beginning of this lesson, possible points of confusion are (1) there are two functions with similar named, `load_tables` and `load_table`, and neither of them actually downloads a table; rather, they download metadata about the tables.
+* Although learners should have seen functions in the SWC python episode, its always good to reiterate the process of prototyping what you want to do. Then if you find yourself wanting to do it over and over, make it into a function (a good rule of thumb is if you are copy and pasting, then think about whether you should be using a function).
 
-* Note that even though we don't use the parallax column for our analysis, we include it in the results for use in the data exploration section
+* Highlight throughout the episode what commands are ADQL specific and what are general SQL commands.
 
-* For `query2` (and most subsequent queries) if you get exactly 2000 results, it's probably because you used `launch_job` rather than `launch_job_async`. This is a good opportunity to remind learners: if you get exactly 2000 results, check that you did an async query and not a synchronous query
+* Errors will happen throughout the workshop. In an online setting it can be especially useful for learners to share their screen with the class while you troubleshoot together. This gives other learners the opportunity to see how you troubleshoot and to see common errors. However, be aware of the time and do not derail the entire episode if someone is having an edge case issue.
+
+* Think about how to communicate exercises to learners, especially in an on line setting. In our first beta workshop both instructors and learners were really happy with [hackmd](https://hackmd.io).
```

Comment:
```suggestion
* Think about how to communicate exercises to learners, especially in an online setting. In our first beta workshop both instructors and learners were really happy with [hackmd](https://hackmd.io).
```

Reply:
I can't tell if this entire block is about the spelling of on line (spelling is not my strong suit) or if there is a larger comment that I'm missing

Reply:
Yes, just on line to online.